shark/shark-android/src/test/java/shark/AndroidObjectInspectorsTest.kt

```diff
@@ -2,12 +2,13 @@ package shark
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
-import shark.LeakTraceObject.LeakingStatus.LEAKING
-import shark.LeakTraceObject.LeakingStatus.NOT_LEAKING
+import shark.LeakTraceObject.LeakingStatus.UNKNOWN
 import shark.LeakTraceObject.ObjectType.INSTANCE
 
 class AndroidObjectInspectorsTest {
 
+  // TODO: There is another trace with the extended WeakHashMap.
```

Comment:
<details>
<summary>Trace before</summary>
<img width="500" alt="image" src="https://github.com/square/leakcanary/assets/37583380/72ac016f-3b79-4424-82ea-d627d1385289">
</details>

<details>
<summary>Trace after</summary>
<img width="500" alt="image" src="https://github.com/square/leakcanary/assets/37583380/c373b412-a06a-487e-b2fc-0ba48ae614ca">
</details>

Reply:
https://github.com/square/leakcanary/issues/2340

Reply:
This is happening because this change changed the expected behavior for weak keys, making them look like strong keys. That won't work.